packages/ckeditor5-block-quote/docs/_snippets/features/nested-block-quote.html

```diff
@@ -0,0 +1,43 @@
+<div id="snippet-nested-block-quote">
+	<h3>The Gagarin Mixup</h3>
+
+	<p>Misquoting is more often than one may think. And it may happen to almost anybody. Sometimes it would be an accident, but sometimes we may consider this a conscious act. One of the better known cases is something a Soviet cosmonaut, Yuri Gagarin, was supposed to say but apparently never said. The infamous quote goes:</p>
+
+	<blockquote>
+	<p>I looked and looked but I didn&#39;t see God.</p>
+	</blockquote>
+
+	<p>Let us refer to the <strong>Yuri Gagarin's</strong> <a href="https://en.wikiquote.org/wiki/Yuri_Gagarin#Disputed">Wikiquote page</a> on that topic and dig further into this particular case study, as it offers a plausible and factual explanation of this mess-up:</p>
+
+	<blockquote>
+	<p>Colonel Valentin Petrov stated in 2006&nbsp;that the cosmonaut never said such words, and that the quote originated from Nikita Khrushchev&#39;s speech at the plenum of the Central Committee of the CPSU about the state&#39;s anti-religion campaign, saying:</p>
+
+	<blockquote>Gagarin flew into space, but didn&#39;t see any god there.</blockquote>
+
+	<p>Gagarin himself was a member of the Russian Orthodox Church.</p>
+	</blockquote>
+
+	<p>It was, apparently, a cunning rhetoric to support an ideological point of view and we may even suppose the cosmonaut himself was not personally involved in it. Quite honestly being a believer he would have hard time denying the existence of God. It is something Gherman Titov, the second Soviet cosmonaut, would rather say, being an atheist.</p>
+
+	<p>It was, in fact, duly noted in <i>The New York TImes</i>, dated May 7, 1962:</p>
+	<blockquote>
+		Maj. Gherman S. Titov, the Soviet astronaut, proclaimed his disbelief in God today. He said he saw
+		<blockquote>no God or angels</blockquote>
+		during his seventeen orbits of the earth.
+	</blockquote>
+
+	<p>A similar statement by Titov was also reported by Pavel Barashev and Yuri Dokuchayev, so it seems this was not entirely made up by Khrushchev, but rather reused and falsely attributed to the Earth's first space farer.</p>
+
+	<p>The Wikiquote webpage offers additional insight into this curious misattribution with a variant quote, practically ruling out the possibility it was Gagarin saying:</p>
```

Comment:
I would cut this third situation out to shorten the demo.

Reply:
Addressed in commit.